src/core/main.cpp

```diff
@@ -510,6 +510,15 @@ int main( int argc, char * * argv )
 		Engine::getSong()->loadProject( file_to_load );
 		printf( "done\n" );
 
+		if ( Engine::getSong()-> hasErrors() )
+		{
+			const char * summary = Engine::getSong()->errorSummary()->toStdString().c_str();
+
+			fputs( "\n", stderr );
+			fputs( summary, stderr );
+			fputs( "\n", stderr );
```

Comment:
I don't understand, this looks like a misleading title for this function. I want to print a warning, to the user, that the current song project has errors, not really debug it.